### DISCUSSION TITLE
CLI-only version

### DISCUSSION THREAD

**Discussion by Tualua** [AdnanHodzic/auto-cpufreq#568] [Q&A]
Hello!
Is there a way to install and use CLI-only version?

**Comment by AdnanHodzic** [comment]
Of course, by default it's CLI: https://github.com/AdnanHodzic/auto-cpufreq#install---auto-cpufreq-daemon

Or for more info see: https://m.youtube.com/watch?v=SPGpkZ0AZVU&list=LL&index=1&t=419s
